.github/workflows/check-sorbet-typing-mode.yml

```
@@ -0,0 +1,57 @@
+name: Sorbet
+on: # yamllint disable-line rule:truthy
+  pull_request:
+  push:
+    branches: [main]  # Adjust this to the branch you want for push events
+
+jobs:
+  check-sorbet-typing-mode:
+    name: Sorbet Typing Mode Check
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+
+    steps:
+      - uses: actions/checkout@v4
```

nit: can you pin his as well
```suggestion
      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

---

FWIW this isn't needed since it's been migrated to an immutable action. NBD either way.